app/views/paye/paye_government_spending.scala.html

```diff
@@ -0,0 +1,118 @@
+    </div>
+
+    @if(lang.language == "cy"){
+        <a id="english-switch" href=@controllers.routes.TaxsLanguageController.switchLanguage("en") data-journey-click="Language:Select:English">@Html(Messages("generic.english"))</a>
+    } else {
+        <a id="welsh-switch" href=@controllers.routes.TaxsLanguageController.switchLanguage("cy") data-journey-click="Language:Select:Welsh">@Html(Messages("generic.welsh"))</a>
+    }
+}
+
+@includes.taxs_main(Messages("ats.treasury_spending.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo), "", contentHeader = Option(headerContent), actingAttorney = actingAsAttorney) {
+
+    @helpers.back_link(Messages("generic.back"))
+
+    <div class="print-this">
+        <img class="print-this__icon" src="/annual-tax-summary/assets/images/icon-print.svg" alt="Print Icon">
+        <a class="print-this__link" href="javascript:window.print()" data-journey-click="@Messages("ats.treasury_spending.print.click"):@Messages("ats.treasury_spending.title"):@Messages("ats.treasury_spending.print.this.page")">@Messages("ats.treasury_spending.print.this.page")</a>
+    </div>
+
+    <h1>
+        @Html(Messages("ats.treasury_spending.title"))<br>
+        @Html(Messages("generic.tax_year_heading", viewModel.taxYearFrom, viewModel.taxYearTo))
+    </h1>
+            @if(viewModel.incomeTaxStatus == "0002" && !viewModel.scottishIncomeTax.isZero) {
+                <p class="lede">@Html(Messages("ats.treasury_spending.section.sit.lede"))</p>
+                <p class="lede">@Html(Messages("ats.treasury_spending.section.sit.scottish.spending.1")) <a href="http://www.gov.scot/accounts" target="_blank" data-journey-click="link - click:@Html(Messages("ats.treasury_spending.html.title")):@Html(Messages("ats.treasury_spending.section.sit.scottish.spending.link.1.text"))">@Html(Messages("ats.treasury_spending.section.sit.scottish.spending.link.1.text"))</a> @Html(Messages("ats.treasury_spending.section.sit.scottish.spending.2")) <a href="http://www.gov.uk/scottish-rate-income-tax/how-it-works" target="_blank" data-journey-click="link - click:@Html(Messages("ats.treasury_spending.html.title")):@Html(Messages("ats.treasury_spending.section.sit.scottish.spending.link.2.text"))">@Html(Messages("ats.treasury_spending.section.sit.scottish.spending.link.2.text"))</a></p>
+            } else {
+                <p class="lede">@Html(Messages("ats.treasury_spending.section.lede"))</p>
+            }
+
+            <table id="tax-spent" aria-hidden="false">
+                <thead>
+                <tr>
+                    <th scope="col">@Html(Messages("generic.description"))</th>
+                    <th scope="col">@Html(Messages("generic.amount_pounds"))</th>
+                </tr>
+                </thead>
+                <tbody>
+
+                @viewModel.govSpendAmountData.toList.filter(  _._1 != "GovSpendTotal").sortWith(_._2.percentage > _._2.percentage).map {
```

Comment:
Hi @oc-ste again, our remit was to duplicate these pages, pop them in a paye specific folder for separation and then update them with paye specific content/data/needs. As far as I'm aware, it wasn't to update and fix tech debt. This code lives in live on SA too - https://github.com/hmrc/tax-summaries-frontend/blob/master/app/views/government_spending.scala.html#L77

Reply:
The responsibility is not to fix the existing tech debt, but also not add to it. I agree that this code is difficult to understand as is right now - please could the ATS team quickly review whether there is a way to easily move this to the view model instead of the view?

If that isn't easy, @oc-ste we might need a discussion about how plausible it is to accept this debt for the time being, please.